Code/GraphMol/ChemReactions/ReactionFingerprints.cpp

```diff
@@ -89,25 +95,43 @@ ExplicitBitVect *generateFingerprintAsBitVect(RDKit::ROMol &mol,
   mol.updatePropertyCache(false);
   ExplicitBitVect *res;
```

Comment:
This is not really required as there is no chance that `res` will be returned undefined, but it may help future-proofing the code in case it changes (you initialized `SparseIntVect *res` to `nullptr` above).
```suggestion
  ExplicitBitVect *res = nullptr;
```